pandas/core/categorical.py

```diff
@@ -1995,14 +1995,18 @@ def is_dtype_equal(self, other):
 
         Parameters
         ----------
-        other : Categorical
+        other : Categorical, Series
 
         Returns
         -------
         are_equal : boolean
         """
-
         try:
+            from pandas.core.series import Series
+
+            if isinstance(other, Series):
```

Comment:
I don't find this necessarily a bug. Personally I would find keeping `ordered=False` as the default much clearer.